azure-pipelines.yml

```
@@ -77,6 +77,14 @@ steps:
     cargo test --verbose --all --features "test unstable"
   displayName: test
 
+- script: |
+    cargo install --verbose cargo-tarpaulin --version '0.13.3'
+    cargo tarpaulin --verbose --all --features "test unstable" --out Xml
```

```suggestion
    cargo tarpaulin --verbose --all --features "test unstable" --out Xml --run-types  Tests Doctests
```

Just got this running locally. Looks like we need the `--run-types` argument because tarpaulin doesn't run doctests by default. (This crate is mostly doctests)